tests/src/Flyvemdm/Tests/CommonTestCase.php

```diff
@@ -186,6 +186,56 @@ protected function createFleet($input) {
       return $fleet;
    }
 
+   /**
+    * @param array $input input data
+    * @return \PluginFlyvemdmTask
+    */
+   protected function createTask($input) {
+      $task = $this->newMockInstance(\PluginFlyvemdmTask::class, '\MyMock');
+      $task->getMockController()->post_addItem = function () {};
+      $taskId = $task->add($input);
+      $this->boolean($task->isNewItem())->isFalse();
+
+      $task = new \PluginFlyvemdmTask();
+      $task->getFromDB($taskId);
+
+      return $task;
+   }
+
+   /**
+    * @param string $policySymbol
+    * @return array
+    */
+   protected function createFleetAndTask($policySymbol = 'storageEncryption') {
+      $fleet = $this->createFleet(['name' => $this->getUniqueString()]);
```

Comment:
Remove this method, the test should call createFleet, then createTask.

Reply:
This method is a from a code refactor used on createAgentTaskstatus and two more test that haven't been published yet because the code is on another PR.

Reply:
When a method has a name telling that ot dies several things, it is better to split it.

The caller could do the following:
```php
$fleet = $this->createFleet();
$task = $this->createTask($fleet);
// test whatever with these 2 above objects
```
```

Reply:
Ok, I approve, and let's see the final result !